### ISSUE TITLE
File cards in search should link to new File page

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#3555]

**Comment by raprasad** [comment]
- [ ] mydata, link file card to file page
   - file: `cards_miniumum.html`
   - line #: 26
   - example update:
```html
            <a href="/file.xhtml?fileId={{ card_info.entity_id }}"><span style="padding:4px 0;">{{ card_info.name }}</span></a>

```

**Comment by mheppler** [comment]
The fix for navigating from the dataverse pg results cards for a file to the file pg has been fixed with the work done in **Customization Header/Footer/Homepage - Optional new landing page for installation #3639**, removing the `ShowFileLandingPage` setting from the search-include and related files.

This issue will remain open, to fix the My Data pg, once the bug in ** MyData: clicking files facet hangs/gives error (added in 4.6.2) #3886** is resolved.

**Comment by raprasad** [comment]
@mheppler : I added the link fixes into #3866.  
  - e.g. The links in the MyData File card now go to the file page.

**Comment by mheppler** [comment]
Closing this issue as delivered in #3866 by @raprasad. 👍
